src/test/java/com/google/cloud/spanner/pgadapter/ProtocolTest.java

```diff
@@ -1202,16 +1202,21 @@ public void testCopyDataMessage() throws Exception {
     MutationWriter mb = Mockito.mock(MutationWriter.class);
     Mockito.when(copyStatement.getMutationWriter()).thenReturn(mb);
 
-    WireMessage message = ControlMessage.create(connectionHandler);
-    Assert.assertEquals(message.getClass(), CopyDataMessage.class);
-    Assert.assertArrayEquals(((CopyDataMessage) message).getPayload(), payload);
+    {
+      WireMessage message = ControlMessage.create(connectionHandler);
+      Assert.assertEquals(message.getClass(), CopyDataMessage.class);
+      Assert.assertArrayEquals(((CopyDataMessage) message).getPayload(), payload);
 
-    CopyDataMessage messageSpy = (CopyDataMessage) Mockito.spy(message);
-    messageSpy.send();
+      CopyDataMessage messageSpy = (CopyDataMessage) Mockito.spy(message);
+      messageSpy.send();
+    }
 
-    Mockito.verify(mb, Mockito.times(1)).buildMutation(connectionHandler, payload);
+    Mockito.verify(mb, Mockito.times(1)).addMutations(connectionHandler, payload);
   }
 
+  @Test
+  public void testMultipleCopyDataMessages() throws Exception {}
```

Comment:
I assume that this is a placeholder for a test that is coming in a next PR?

Reply:
Updated now.